src/fr/articles/landmarks-aria/index.njk

```diff
@@ -2,136 +2,207 @@
-  <li>La balise <code>footer</code> ou le <code>role=contentinfo</code> permettent d’identifier le pied de page. Il doit être unique dans la page, sauf lorsqu'il est présent dans des balises <code>section</code>, <code>role=region</code> ou <code>article</code>, <code>role=article</code>.</li>
-  <li>La balise <code>header</code> ou <code>role=header</code> identifie l’en-tête de la page. Lui aussi doit être unique, sauf lorsqu'il est présent dans des balises <code>section</code>, <code>role=region</code> ou <code>article</code>, <code>role=article</code>.</li>
-  <li>Pour identifier le formulaire de recherche dans le site, on utilisera seul le <code>role=search</code> <abbr>ARIA</abbr>.</li>
-  <li>Si besoin, le contenu annexe au contenu principal de la page (information complémentaire, publicité…) doit être identifié en utilisant la balise <code>aside</code> ou le <code>role=complementary</code>.</li>
+<p>Pour ajouter de la sémantique HTML au contenu d'une page, 2 possibilités :</p>
+<ol>
+  <li>
+    <p><u>Pour une page <em>en HTML5</em></u> (recommandé) : utiliser les <em>balises sémantiques </em>(<code>main</code>, <code>header</code>, <code>footer</code>, <code>aside</code>, ...) qui y ont été introduites.
+    Ces balises doivent être utilisées seules, sans les rôles <abbr>ARIA</abbr> correspondants, car leur support est bon dans la grande majorité des navigateurs et des aides techniques.</p>
+
+    <p>Exemple&nbsp;:</p>
+    <p class="bg-primary d-inline-block p-2 mb-0 font-weight-bold">HTML</p>
+    <pre class="mb-0"><code class="html">
+    &lt;head&gt;
+      […]
+      &lt;title&gt;Titre de la page&lt;/title&gt;
+      […]
+    &lt;/head&gt;
+
+    &lt;body&gt;
+      &lt;header&gt;
+        &lt;img&gt;logo&lt;/img&gt;
+        &lt;h1&gt;Nom de l'application&lt;/h1&gt;
+        &lt;nav&gt;…navigation…&lt;/nav&gt;
+        […]
+      &lt;/header&gt;
+
+      &lt;main&gt;
+        &lt;nav aria-label="Fil d’Ariane"&gt;[…]&lt;/nav&gt;
+        &lt;section&gt;
+          &lt;h1&gt;Actualités&lt;/h1&gt;
+          &lt;p&gt; 21 juin 2018 - Fête de la musique&lt;/p&gt;
+          &lt;p&gt;…contenu de la section…&lt;/p&gt;
+        &lt;/section&gt;
+        &lt;article&gt;
+          […]
+        &lt;/article&gt;
+        […]
+      &lt;/main&gt;
+
+      &lt;aside&gt;
+        &lt;p&gt;…contenu annexe en relation avec le contenu principal…&lt;/p&gt;
+        […]
+      &lt;/aside&gt;
+
+      &lt;footer&gt;
+        &lt;p&gt;…contenu du pied de page&nbsp;: copyright…&lt;/p&gt;
+        […]
+      &lt;/footer&gt;
+    &lt;/body&gt;
+    </code></pre>
+    <br>
+  </li>
+
+  <li>
+    <p></p><u>Pour une page qui n'est <em>pas en HTML5</em></u> : ajouter des <em>régions (<i lang="en">landmarks</i>) <abbr>ARIA</abbr></em></p>
```

Comment:
To stay consistent with the reste of the website, underlined text is for links only.